scripts/categories/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/gocarina/gocsv"
+	"gopkg.in/yaml.v2"
+)
+
+type Category struct {
+	Key  string `csv:"key"`
+	Name string `csv:"name"`
+	ID   string `csv:"id"`
+}
+
+func createCategory(c []*Category) {
+	wd, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+	dirPath := filepath.Join(wd, "data")
+	err = os.MkdirAll(dirPath, os.ModePerm)
+	if err != nil {
+		panic(err)
+	}
+	f, err := os.OpenFile(filepath.Join(dirPath, "categories.yml"), os.O_WRONLY|os.O_CREATE, os.ModePerm)

```

Comment:
前回抜けてたのですが、これが必要そうでした 🙏 
```suggestion
	f, err := os.OpenFile(filepath.Join(dirPath, "categories.yml"), os.O_WRONLY|os.O_CREATE|os.O_TRUNC, os.ModePerm)

```

Reply:
こちら参考にしました https://stakiran.hatenablog.com/entry/2019/06/19/071707